src/api.js

```diff
@@ -112,6 +112,18 @@ export const deleteOrder = (orderId) => {
   );
 };
 
+/************************* REVIEWS *****************************/
+
+export const createReview = (review) => {
```

Comment:
I think it would be great if you create a service for that with a try/catch in this case

Reply:
But i do the try and catch in the place i use it. I dont know if I understand the benefits, can you explain.